addons/portal/static/src/js/portal_chatter.js

```diff
@@ -307,7 +307,15 @@ publicWidget.registry.portalChatter = publicWidget.Widget.extend({
         var defs = [this._super.apply(this, arguments)];
         var chatter = new PortalChatter(this, this.$el.data());
         defs.push(chatter.appendTo(this.$el));
-        return Promise.all(defs);
+        return Promise.all(defs).then(function () {
+            // scroll to the right place after chatter loaded
+            if (window.location.hash === '#discussion') {
```

Comment:
'discussion' should not be hardcoded here. Is not it equal to `this.$el.attr('id')` ?

Reply:
Ok